src/llama_stack_provider_ragas/remote/provider.py

```diff
@@ -0,0 +1,31 @@
+from llama_stack.providers.datatypes import (
+    AdapterSpec,
+    Api,
+    ProviderSpec,
+    remote_provider_spec,
+)
+
+
+def get_provider_spec() -> ProviderSpec:
+    return remote_provider_spec(
+        api=Api.eval,
+        adapter=AdapterSpec(
```

Comment:
a note. When you bump to 0.2.23+ this will break: https://github.com/llamastack/llama-stack/pull/3378